### ISSUE TITLE
Arguments passed not correctly

### ISSUE THREAD

**Issue by Deilan** [remy/nodemon#1043]
Using the following `nodemon.json`:
```json
{
    "exec": "ts-node {{pwd}}/src/index.ts"
}
```

Attempting to launch the following command `nodemon --inspect=5858` or `nodemon --inspect=5858 --config nodemon.json` doesn't pass `--inspect=5858` to Node.js, but passes it to ts-node at the end:
```
C:\Dev\node-typescript-vscode>nodemon --inspect=5858
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `ts-node C:\Dev\node-typescript-vscode/src/index.ts --inspect=5858`
Hello, world
[nodemon] clean exit - waiting for changes before restart
```

**Comment by Deilan** [comment]
@remy This is not stale.

**Comment by remy** [comment]
This _is_ being passed correctly, as defined by your `exec`.

I think what you need here is a feature request where by you can slot in the `execArgs` between your `ts-node` and arguments. i.e. something like:

```json
{
    "exec": "ts-node {{execArgs}} {{pwd}}/src/index.ts"
}
```

I'm going to close as it can be worked around (if it's a single case use), or would happily take a PR to add this functionality (possibly mapping any nodemon option into the strings?).

**Comment by Deilan** [comment]
For reference: it's been fixed in #1142.
